cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -78,30 +78,31 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
 
 #. Add additional entries to the YARN Application Classpath for Spark jobs.
 
-   .. highlight:: console
-
    If you plan on running Spark programs from CDAP, CDAP requires that additional entries be added to
    the YARN application classpath, as the Spark installed on Cloudera Manager clusters
    is a "Hadoop-less" build and does not include Hadoop jars required by Spark.
 
-   To resolve this, go to the CM page for your cluster, click on the YARN service, then click on
-   the configuration tab, and then enter "mapreduce.application.classpath" in the search box. You will see entries similar to these:
-
-    $HADOOP_MAPRED_HOME/*
+   To resolve this, go to the CM page for your cluster, click on the YARN service, click on
+   the configuration tab, and then enter ``mapreduce.application.classpath`` in the search box.
+   You will see entries similar to these::
 
-    $HADOOP_MAPRED_HOME/lib/*
+     $HADOOP_MAPRED_HOME/*
 
-    $MR2_CLASSPATH
+     $HADOOP_MAPRED_HOME/lib/*
 
+     $MR2_CLASSPATH
 
-   Copy all the entries to "yarn.application.classpath" configuration
-   for Yarn on your Cluster. The "yarn.application.classpath" setting can be found the similar way  mentioned above.
+   Copy all the entries to the ``yarn.application.classpath`` configuration for YARN on your Cluster. 
+   The ``yarn.application.classpath`` setting can be found by searching as mentioned above.
 
-   Add the entries required by scrolling to the last entry in the classpath form, clicking the "+" button to add a new text box entry field at the end. Once you have added all the entries from "mapreduce.application.classpath" to "yarn.application.classpath click on Save.
+   Add the entries required by scrolling to the last entry in the class path form,
+   clicking the "+" button to add a new text box entry field at the end. Once you have
+   added all the entries from the ``mapreduce.application.classpath`` to the
+   ``yarn.application.classpath``, click on *Save*.
 
-   You can make these changes `using Cloudera Manager
-   <http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cm_mc_mod_configs.html>`__.
-   You will be prompted to restart the stale services after making changes.
+You can make these changes `using Cloudera Manager
+<http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cm_mc_mod_configs.html>`__.
+You will be prompted to restart the stale services after making changes.
```

Comment:
Would it be better to say 'Please restart the stale services upon seeing a prompt to do so after making the above changes'?